### ISSUE TITLE
Microsoft Office 2021 applications exit shortly after opening a file while capemon is running

### ISSUE THREAD

**Issue by seanthegeek** [kevoreilly/CAPEv2#1595]
CAPE will launch the Office application, and the file will open, but then close a second later if capemon is monitoring, even if human simulation is disabled. I'm thinking Office must be crashing.

I know that the CAPE documentation [recommends](https://capev2.readthedocs.io/en/latest/installation/guest/requirements.html#additional-software) using Microsoft Office 2010 SP2, but I don't have that version, and it is very difficult, if not impossible to find untainted ISOs for old versions of Microsoft Office.

Thankfully, I have a copy of Office 2016, which works well with CAPE. Other users might not be so lucky.

**Comment by kevoreilly** [comment]
Yep I hate to be harsh but that version of office is 'not supported'.

**Comment by kevoreilly** [comment]
I will however update the documentation to reflect that Office 2016 is another version that should work.

**Comment by kevoreilly** [comment]
Docs updated: https://github.com/kevoreilly/CAPEv2/commit/8c57f5505a6c52d9114e04abb53191e9c31777e1

**Comment by seanthegeek** [comment]
Totally get it. Just wanted to document it. I'll try other versions too so I can build a list of what works and what doesn't.

I'm also going to work on making the signatures less noisy for modern versions of office. I've got a personal todo list of PRs I want to do eventually, ranging from signatures to documentation. I just don't know anything about Windows internals (or C++ really - Python is my jam) to help at all with `capemon`. That said, I'd love to learn if either of you have pointers on where to start.

**Comment by doomedraven** [comment]
all the contributions are more than welcome. My personal policy here is that if i don't use that feature, i don't care about it :D the good open source life ahahahah
